python/cuml/neighbors/kneighbors_regressor.pyx

```diff
@@ -231,3 +231,9 @@ class KNeighborsRegressor(NearestNeighbors, RegressorMixin):
 
     def get_param_names(self):
         return super().get_param_names() + ["weights"]
+
+    def _more_tags(self):
+        return {
+            # fit and predict require conflicting memory layouts
+            'preferred_input_order': None
```

Comment:
I looked more closely at the kneighbors variants and they do actually require `order='F'` as input. We can safely set this `F` here and in the `kneighbors_classifier`.